scrapy/robotstxt.py

```diff
@@ -40,17 +55,7 @@ class PythonRobotParser(RobotParser):
     def __init__(self, robotstxt_body, spider):
         from six.moves.urllib_robotparser import RobotFileParser
         self.spider = spider
-        try:
-            robotstxt_body = to_native_str(robotstxt_body)
-        except UnicodeDecodeError:
-            # If we found garbage or robots.txt in an encoding other than UTF-8, disregard it.
-            # Switch to 'allow all' state.
-            logger.warning("Failure while parsing robots.txt using %(parser)s."
-                           " File either contains garbage or is in an encoding other than UTF-8, treating it as an empty file.",
-                           {'parser': "RobotFileParser"},
-                           exc_info=sys.exc_info(),
-                           extra={'spider': self.spider})
-            robotstxt_body = ''
+        robotstxt_body = decode_robotstxt(robotstxt_body, spider, True)
```

Comment:
could you please use keyword argument to pass boolean value (True), to make code more readable - here and in other places?